Add meta description to some pages (#3853)
wazuh/wazuh-documentation: source/user-manual/registering/password-authorization-registration.rst

.. Copyright (C) 2021 Wazuh, Inc.
.. meta::

>   :description: Check out more about registering Wazuh agents and the registration service with password authorization in our documentation. 

Reviewer: ```suggestion
  :description: Check out our documentation to learn about registering Wazuh agents and the registration service with password authorization. 
```
